pkg/rehearse/jobs.go

```diff
@@ -159,6 +166,11 @@ func filterPeriodics(changedPeriodics []prowconfig.Periodic, allowVolumes bool,
 	for _, periodic := range changedPeriodics {
 		jobLogger := logger.WithField("job", periodic.Name)
 
+		if err := validateLabels(periodic.Labels); err != nil {
```

Comment:
It's OK if someone does not opt into rehearsal. These are not really errors -- we should just 


```go
	if value, ok := labels[CanBeRehearsedLabel]; !ok || value != "true" {
		continue
	}
```

Reply:
This is what am doing here. Check `validateLabels()`

Reply:
I mean continue, not raise an error

Reply:
addressed